src/reportSources/legacyFflogs/eventAdapter/translate.ts

```diff
@@ -198,6 +198,7 @@ export class TranslateAdapterStep extends AdapterStep {
 
 		// Instant kills don't seem to include a sequence, so we're faking it
 		const sequence = this.nextFakeSequence--
+		const bonusPercent = event.bonusPercent || 0
```

Comment:
```suggestion
		const bonusPercent = event.bonusPercent ?? 0
```
We prefer ?? for nullish coalescence, rather than relying on falsey behavior.